.gitignore

```diff
@@ -159,4 +159,6 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
-.dbsecret.json
\ No newline at end of file
+.dbsecret.json
+
+notes.txt
```

Comment:
try to keep unrelated files out of your git repository. 
only include/exclude files that are common across all developers